javascript/src/main/components/Channels/ChannelTable.js

```diff
@@ -10,6 +10,20 @@ const channelLink=(channelName) => {
 
 }
 
+
+const textLinks = (text) => {
+    var bracketRegEx = /<(http(?:.*?))>/g;
```

Comment:
What happens if there are multiple links in the purpose or topic? There are no current cases for this in the current slack but this could be a possible case. 

Note: I tried to make a test channel so you there would be a case for you to test for, but I found out syncing the slack channels to the app has some issues. So just make a new test for multiple links and make sure it passes

Reply:
Just added a test for that!